src/integrationtest/java/com/capgemini/brahma/examples/route/RouteIntegrationTests.java

```diff
@@ -1,21 +1,25 @@
 package com.capgemini.brahma.examples.route;
 
-import com.capgemini.brahma.Application;
-import com.jayway.restassured.RestAssured;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.test.SpringApplicationConfiguration;
+import org.springframework.boot.test.WebIntegrationTest;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
-import org.springframework.test.context.web.WebAppConfiguration;
+
```

Comment:
can you move these back up? we only separate out project-local code down to this block